t/test.c

```diff
@@ -495,10 +495,10 @@ int main(int argc, char **argv)
     subtest("test-vector", test_vector);
     subtest("test-retry-aead", test_retry_aead);
     subtest("transport-parameters", test_transport_parameters);
+    subtest("cid", test_cid);
```

Comment:
Just curious, is there a particular reason for this ordering change?

Reply:
The tests are listed in the order of their scope. We start from unit tests of submodules, start doing more things, and finally end up testing the whole machinery. Because the latter is built upon the submodules assuming that they work as expected.

`test_cid` is testing submodules, therefore placed before other tests that uses the submodules being tested.

Reply:
Makes sense, thank you for explanation!